subsys/bluetooth/mesh/Kconfig

```diff
@@ -814,10 +814,17 @@ config BT_MESH_SEQ_STORE_RATE
 	  off with a value that's guaranteed to be larger than the last
 	  one used before power off.
 
+config BT_MESH_RPL_STORE
+	bool "Enable storing RPL entries"
+	default y
+	help
+	  Enable storing RPL in persistent memory using settings subsystem.
+
 config BT_MESH_RPL_STORE_TIMEOUT
 	int "Minimum interval after which unsaved RPL entries are updated in storage"
 	range -1 1000000
-	default 5
+	default 5 if BT_MESH_RPL_STORE
+	default -1
```

Comment:
Is this really necessary? Because it BT_MESH_RPL_STORE is set to "n" then this value does not have any effect. So if it is 5 or -1 does not matter. So this could just always be default 5.

Reply:
Could also have a `depends on BT_MESH_RPL_STORE` instead of defaulting to `-1`